build-redhat.sh

```diff
@@ -21,6 +21,8 @@ FIREHOL_MD5=`cut -f1 -d' ' < build/firehol.md5`
 IPRANGE_MD5=`cut -f1 -d' ' < build/iprange.md5`
 NETDATA_MD5=`cut -f1 -d' ' < build/netdata.md5`
 
+IPRANGE_RELEASE="1"
```

Comment:
You are correct; the release number is hardcoded in the .spec file. It needs updating when a new release of the same firehol RPM is made. See https://www.thegeekdiary.com/understanding-rpm-versions-and-naming-schemes/

Since the build will fail if they are out of line, I'm OK with this approach. If it were me I would probably have determined IPRANGE_RELEASE dynamically here, or perhaps have just wildcarded the RPM name to be installed: Travis builds always go from clean, so there will be only one match.

```suggestion
IPRANGE_RELEASE="2"
```